tests/network/network_policy/test_network_policy.py

```diff
@@ -54,24 +54,30 @@ def namespace_2(admin_client, unprivileged_client):
 
 
 @pytest.fixture()
-def deny_all_http_ports(namespace_1):
-    with ApplyNetworkPolicy(name="deny-all-http-ports", namespace=namespace_1.name) as np:
+def deny_all_http_ports(namespace_1, unprivileged_client):
+    with ApplyNetworkPolicy(name="deny-all-http-ports", namespace=namespace_1.name, client=unprivileged_client) as np:
         yield np
 
 
 @pytest.fixture()
-def allow_all_http_ports(namespace_1):
+def allow_all_http_ports(namespace_1, unprivileged_client):
```

Comment:
please follow https://github.com/RedHatQE/openshift-virtualization-tests/blob/main/docs/CODING_AND_STYLE_GUIDE.md?plain=1#L73 in terms of fixture ordering